### PR TITLE
Updates documentation for docker over public networks

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: docs/installation_guide.rst [hunk 1]
```diff
@@ -268,7 +269,117 @@ data will be unavailable.
...
+
+With this  ``traefik`` will act as a proxy with basic authentication that will protect your rotki instance interface
+from unauthorized access. When navigating to ``http://rotki.example.com`` you will get a browser popup where you
+have to insert the ``AUTH_USER`` and the password you created with ``htpasswd``.
+
+After passing the base authentication step you will be greeted with the familiar login page of rotki.
+
+.. note::
+
+    As a bonus you should not be able to also access rotki from your mobile phone.
```
Comment (yabirgb) [inline docs/installation_guide.rst:378]:
```suggestion
    As a bonus you should now be able to also access rotki from your mobile phone.
```

File: docs/installation_guide.rst [hunk 2]
```diff
@@ -268,7 +269,117 @@ data will be unavailable.
...
+If you prefer to use docker compose, a docker-compose.yml template is provided below for convenience.
+
+Using docker over a public network
+----------------------------------
+
+Since rotki was designed to run over a public network it is :ref:`generally advised <docker warning>` to avoid
+directly exposing your rotki instance over public network.
+
+If you still want to access rotki over a public network it is suggested to use at least a proxy with
+basic authentication and an intermediate layer of security. With this setup anyone that visits your public domain
```
Comment (yabirgb) [inline docs/installation_guide.rst:281]: isn't the proxy with basic auth the intermediate layer of security?

**Review by yabirgb — COMMENTED** [review]
One typo and one question. Also I'm getting this

```
/home/yabirgb/work/rotki/docs/usage_guide.rst:218: WARNING: Title underline too short.

Use abbreviations for large numbers.
^^^^^^^^^^^^^^^^^^^^
/home/yabirgb/work/rotki/docs/usage_guide.rst:218: WARNING: Title underline too short.

Use abbreviations for large numbers.
^^^^^^^^^^^^^^^^^^^^
```

File: docs/installation_guide.rst (hunk 2 above)
Comment (kelsos) [reply docs/installation_guide.rst:281]: /and/as/

**Comment by kelsos** [comment]
> One typo and one question. Also I'm getting this
> 
> ```
> /home/yabirgb/work/rotki/docs/usage_guide.rst:218: WARNING: Title underline too short.
> 
> Use abbreviations for large numbers.
> ^^^^^^^^^^^^^^^^^^^^
> /home/yabirgb/work/rotki/docs/usage_guide.rst:218: WARNING: Title underline too short.
> 
> Use abbreviations for large numbers.
> ^^^^^^^^^^^^^^^^^^^^
> ```

I have no idea what this is and I am pretty sure I didn't touch 218, I did some changes there but they didn't work so I gave up 😛

**Review by yabirgb — APPROVED** [review]
LGTM. Thanks sir

**Pull request merged**